### PR TITLE
Edit "Enable HVCI using Intune" section

### PR DESCRIPTION
**Pull request by gduarte14**
## Description
It changes the Intune configuration section of this article from pointing to Applocker CSP to point to Policy CSP - Virtualization Based Technology.

## Why

Whilst reading through documentation and trying to find the location to configure HVCI in Intune I came across this article and also the Policy CSP one. I think the reference to VirtualizationBasedTechnology I am making is more accurate than the Applocker CSP.

## Changes

From this:
"Enabling in Intune requires using the Code Integrity node in the [AppLocker CSP](https://learn.microsoft.com/en-us/windows/client-management/mdm/applocker-csp)."

To this:
"Enabling HVCI via Intune requires configuring the [Policy CSP - VirtualizationBasedTechnology](https://learn.microsoft.com/en-us/windows/client-management/mdm/policy-csp-virtualizationbasedtechnology)."

### CODE REVIEW

File: windows/security/threat-protection/device-guard/enable-virtualization-based-protection-of-code-integrity.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ HVCI is labeled **Memory integrity** in the Windows Security app and it can be a
 
 ### Enable HVCI using Intune
 
-Enabling in Intune requires using the Code Integrity node in the [AppLocker CSP](/windows/client-management/mdm/applocker-csp).
+Enabling HVCI via Intune requires configuring the [Policy CSP - VirtualizationBasedTechnology](https://learn.microsoft.com/en-us/windows/client-management/mdm/policy-csp-virtualizationbasedtechnology).
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/device-guard/enable-virtualization-based-protection-of-code-integrity.md:53]:
```suggestion
Enabling HVCI via Intune requires configuring the [Policy CSP - VirtualizationBasedTechnology](https://learn.microsoft.com/windows/client-management/mdm/policy-csp-virtualizationbasedtechnology).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @gduarte14, kindly commit the suggested change, please. Many thanks.

**Comment by jgeurten** [comment]
@vinaypamnani-msft can you add @rotruxal to review instead. He owns HVCI

File: windows/security/threat-protection/device-guard/enable-virtualization-based-protection-of-code-integrity.md (hunk 1 above)
Comment (gduarte14) [reply windows/security/threat-protection/device-guard/enable-virtualization-based-protection-of-code-integrity.md:53]: Just tested this in Intune with a configuration profile using Settings Catalog and choosing this Configuration settings:
image

This enabled Memory Integrity on my AADJ device: 
image

**Comment by vinaypamnani-msft** [comment]
This was changed in another PR, closing this PR.

**Pull request closed without merging**